dictionaries/python/src/data_science.txt

```diff
@@ -94,3 +94,27 @@ upsample
 parallelizable
 ONNX
 enwik
+Hadamard
+Schur product
+rsqrt
+bucketizes
+tera
+strided
+invvar
+fastai
+behaviour
+nesterov
+sutskever
```

Comment:
Since these are people, maybe we should capitalize proper names. I wonder if it better to add a "people" dictionary. 🤔